src/Uno.UI/Extensions/ViewHelper.Android.cs

```diff
@@ -303,11 +303,18 @@ public static RectF PhysicalToLogicalPixels(this RectF size)
 		[MethodImpl(MethodImplOptions.AggressiveInlining)]
 		public static Rect LogicalToPhysicalPixels(this Rect size)
 		{
+			var physicalBottom = LogicalToPhysicalPixels(size.Bottom);
+			var physicalRight = LogicalToPhysicalPixels(size.Right);
+
+			var physicalX = LogicalToPhysicalPixels(size.X);
+			var physicalY = LogicalToPhysicalPixels(size.Y);
+
+
```

Comment:
```suggestion
			// We convert bottom and right to physical pixels and then determine physical width and height from them, rather than the other way around, to ensure that adjacent views touch (otherwise there can be a +/-1-pixel gap due to rounding error, in the case of non-integer logical dimensions).
```